formatter/stylish.go

```diff
@@ -66,6 +62,10 @@ func (*Stylish) Format(failures <-chan lint.Failure, config lint.Config) (string
 		output += table(val) + "\n"
 	}
 
+	ps := "problems"
+	if total == 1 {
+		ps = "problem"
+	}
 	suffix := fmt.Sprintf(" %d %s (%d errors) (%d warnings)", total, ps, totalErrors, total-totalErrors)
```

Comment:
It seems strange you can have 

"1 problem (1 errors) (0 warnings)"
Or

"1 problem (0 errors) (1 warnings)"

You should also handle the singular/plural for "error" and "warning"

yes it's old code unaffected by the PR, but I think you should fix it